backend/de.metas.acct.base/src/main/java/de/metas/acct/gljournal_sap/process/SAP_GLJournal_ReverseDocument.java

```diff
@@ -74,12 +72,15 @@ public ProcessPreconditionsResolution checkPreconditionsApplicable(final @NonNul
 	@Override
 	protected String doIt()
 	{
-		final SAPGLJournal createdJournal = glJournalService.copy(SAPGLJournalCopyRequest.builder()
+		final SAPGLJournal createdJournal = glJournalService.reverse(SAPGLJournalCopyRequest.builder()
```

Comment:
if `SAPGLJournalCopyRequest` is now only about reversing, it should be named properly

Reply:
done

Reply:
maybe I'm missing something, but I see `SAPGLJournalCopyRequest` was not renamed

Reply:
SAPGLJournalCopyRequest is now renamed